<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1703</width>
    <height>447</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>rc文件转ui文件</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="6">
         <widget class="QPushButton" name="selectAndHightPB">
          <property name="text">
           <string>选择</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>打开</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>查找</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>关键字:BEGIN;END</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="_filepath">
          <property name="text">
           <string>NULL</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_compile">
        <property name="text">
         <string>打开并编译ui</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>修改处</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_clearTextedit">
          <property name="text">
           <string>清理文本</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonConvert">
          <property name="text">
           <string>识别rc转为Qt控件关键参数</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButton_clearTablewidget">
          <property name="text">
           <string>清理表格</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_exportSingleUI">
          <property name="text">
           <string>输出单个UI(没写)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_insertCurrent">
          <property name="text">
           <string>当前行下插入</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_deleteCurrent">
          <property name="text">
           <string>删除当前</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_check">
          <property name="text">
           <string>check</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_export">
          <property name="text">
           <string>输出所有UI</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QTextEdit" name="textEditRcKeyContent"/>
      </item>
      <item>
       <widget class="QTextEdit" name="textEditBindWidgets"/>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <column>
         <property name="text">
          <string>类型</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>ID</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>文本</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>x</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>y</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>width</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>height</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
